From 35d010afddf44b17197d4ecaa68e17e401a860ab Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@st.com>
Date: Tue, 23 Oct 2018 14:38:55 +0200
Subject: [PATCH 39/97] ARM: dts: stm32: add RTC wakeup interrupt on
 stm32mp157c

Add missing RTC wakeup interrupt on stm32mp157c, otherwise, RTC won't be
able to wake up the system.

Change-Id: I6f3e69b34ef91d6b8c23caa35c22dd95919d2971
Signed-off-by: Amelie Delaunay <amelie.delaunay@st.com>
Reviewed-on: https://gerrit.st.com/109447
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
Reviewed-by: CIBUILD <smet-aci-builds@lists.codex.cro.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@st.com>
---
 arch/arm/boot/dts/stm32mp157c.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/stm32mp157c.dtsi b/arch/arm/boot/dts/stm32mp157c.dtsi
index 9018cac13f7cd..20f85bd5303dc 100644
--- a/arch/arm/boot/dts/stm32mp157c.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c.dtsi
@@ -1516,7 +1516,8 @@
 			reg = <0x5c004000 0x400>;
 			clocks = <&rcc RTCAPB>, <&rcc RTC>;
 			clock-names = "pclk", "rtc_ck";
-			interrupts = <GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts-extended = <&intc GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>,
+					      <&exti 19 1>;
 			status = "disabled";
 		};
 
-- 
2.27.0

